src/w2grid.js

```diff
@@ -5203,7 +5203,15 @@
                 _dragData.pressed = true;
 
                 _dragData.timeout = setTimeout(function(){
-                    if ( !_dragData.pressed ) return;
+                    // When dragging a column for reordering, a quick release and a secondary
```

Comment:
I think it is too excessive use of comments. If you would like to add so much comments, create an issue on github and put a link to the issue in comments. I will remove them later on.